.github/workflows/csharp-cd.yml

```diff
@@ -0,0 +1,472 @@
+                      ${{ matrix.host.TARGET }}
+
+            - name: Setup NuGet.exe for use with actions
+              uses: NuGet/setup-nuget@v1.0.5
+              with:
+                  nuget-version: latest
+
+            - name: Restore dependencies - Bash
+              if: ${{ matrix.build.TARGET != 'x86_64-pc-windows-msvc' }}
+              shell: bash
+              run: |
+                  dotnet restore $DOTNET_PROJECT_PATH
+
+            - name: Restore dependencies - Powershell
+              if: ${{ matrix.build.TARGET == 'x86_64-pc-windows-msvc' }}
+              shell: pwsh
+              run: |
+                  dotnet restore $env:DOTNET_PROJECT_PATH
+
+            - name: Build - Bash
+              if: ${{ matrix.build.TARGET != 'x86_64-pc-windows-msvc' }}
+              shell: bash
+              run: dotnet build --no-restore --configuration Release $DOTNET_PROJECT_PATH
+
+            - name: Build - Powershell
+              if: ${{ matrix.build.TARGET == 'x86_64-pc-windows-msvc' }}
+              shell: pwsh
+              run: dotnet build --no-restore --configuration Release $env:DOTNET_PROJECT_PATH
+
+            - name: Pack - Bash
+              if: ${{ matrix.build.TARGET != 'x86_64-pc-windows-msvc' }}
+              shell: bash
+              run: dotnet pack --no-build --configuration Release $DOTNET_PROJECT_PATH
+
+            - name: Pack - Powershell
+              if: ${{ matrix.build.TARGET == 'x86_64-pc-windows-msvc' }}
+              shell: pwsh
+              run: dotnet pack --no-build --configuration Release $env:DOTNET_PROJECT_PATH
+
+            - name: Prepare file matrix for upload
+              id: set-artifact-matrix
+              shell: bash
+              run: |
+                  # Initialize an empty array for JSON objects
+                  declare -a json_objects=()
+
+                  # Loop through .nupkg files
+                  for file in csharp/**/*.nupkg csharp/**/*.snupkg; do
+                    # Check if file exists (to handle case when no matches are found)
+                    if [ -f "$file" ]; then
+                      filename=$(basename "$file")
+                      json_objects+=("{\"file\":\"$file\",\"filename\":\"$filename\"}")
+                    fi
+                  done
+
+                  # Combine all JSON objects into an array and save to output
+                  if [ ${#json_objects[@]} -eq 0 ]; then
+                    echo "files=[]" >> "$GITHUB_OUTPUT"
+                  else
+                    json_array="[$(IFS=,; echo "${json_objects[*]}")]"
+                    echo "files=$json_array" >> "$GITHUB_OUTPUT"
+                  fi
+
+            - name: Upload artifacts
+              uses: actions/upload-artifact@v4
+              with:
+                  name: Asset - ${{ matrix.build.TARGET }}
+                  path: |
+                      csharp/**/*.nupkg
+                      csharp/**/*.snupkg
```

Comment:
We reference the symbols package here, but we are not actually building the symbols package in any of our projects. We should consider updating our project to output the symbols which are valuable for dev debugging.
https://learn.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg

Reply:
Good catch. Added the (forgotten) snupkg configuration to csproj